encoding/protobuf/testing/benchmark_test.go

```diff
@@ -50,13 +52,29 @@ func BenchmarkIntegrationYARPC(b *testing.B) {
 	}
 }
 
-func BenchmarkIntegrationGRPC(b *testing.B) {
+func BenchmarkIntegrationGRPCClient(b *testing.B) {
 	benchmarkForTransportType(b, testutils.TransportTypeGRPC, func(clients *exampleutil.Clients) error {
 		benchmarkIntegrationGRPC(b, clients.KeyValueGRPCClient, clients.ContextWrapper)
 		return nil
 	})
 }
 
+func BenchmarkIntegrationGRPCAll(b *testing.B) {
+	server := grpc.NewServer()
+	examplepb.RegisterKeyValueServer(server, example.NewKeyValueYARPCServer())
+	listener, err := net.Listen("tcp", "0.0.0.0:1234")
+	if err != nil {
+		b.Fatal(err.Error())
+	}
+	go func() { _ = server.Serve(listener) }()
```

Comment:
Is there an error that we should check for and `b.Fatal` on?

Reply:
Per https://godoc.org/google.golang.org/grpc#Server.Serve, "Serve will return a non-nil error unless Stop or GracefulStop is called." We're technically calling stop, but then the semantics get messy to check everything, and checking for errors in the workflow isn't really the point of this benchmarking function.

Tldr: you generally ignore this error when doing simple tests/benchmarks like this

Reply:
Fair enough, I buy it.